### DISCUSSION TITLE
Detection feature for Wyze Cam

### DISCUSSION THREAD

**Discussion by benjani85** [mrlt8/docker-wyze-bridge#399] [Ideas]
Hi,

Just wondering if there are any plans to add detection support for Wyze Cam? I am currently using the Wyze app to receive any detection alerts but it would be great if I can integrate it via Wyze Bridge.

Thanks
-B

**Comment by mrlt8** [comment]
I would love to add some sort of native motion detection from the camera hardware, and have been looking into this from the start.
Unfortunately, it doesn't seem like the cameras expose any sort of motion flag over the TUTK protocol.

I believe the cameras communicate with wyze over secure MQTT via AWS IOT Core, so if we could connect to that, we might be able to better monitor control and monitor any events. 

In the meantime, you could feed the rtsp stream into a motion detection system like [frigate](https://frigate.video) or use the opencv plugin in [scrypted](https://github.com/koush/scrypted) to get notifications in homekit.

**Reply by benjani85** [reply]
that's a shame but thanks for the explanation. Really appreciate your work :)
